src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.nuspec

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design</id>
+    <version>$Version$</version>
+    <authors>Microsoft</authors>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Command line tool to precompile Razor views for a project to improve startup time.</description>
+    <tags>dotnet,razor,views</tags>
+    <serviceable>true</serviceable>
+  </metadata>
+  <files>
+    <file src="buildCrossTargeting/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets" target="buildCrossTargeting/" />
+    <file src="build/*/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.targets" target="build/" />
+    <file src="bin/$configuration$/*/publish/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.Design.dll" target="build/" />
```

Comment:
The * represented the TFM name if that's what you were asking, I've removed the star in favor of being specific about the TFM now.